lib/cdo/script_config.rb

```diff
@@ -16,4 +16,19 @@ def self.allows_public_caching_for_script(script_name)
   def self.cached_scripts
     HttpCache.cached_scripts
   end
+
+  # Returns the names of the scripts affected by the 'Scale Mode' feature setting.
+  def self.scale_mode_scripts
+    cached_scripts + %w(
+      20-hour
+      course1
+      course2
+      course3
+      course4
+      playlab
+      artist
+      infinity
+      iceage
```

Comment:
Are we meant to include all Hour of Code scripts? What about mc, minecraft, starwars, starwarsblocks, frozen, hourofcode, gumball, flappy?